docs/products/kafka/howto/kcat.rst

```diff
@@ -0,0 +1,112 @@
+Use kcat with Aiven for Apache Kafka
+====================================
+
+`kcat <https://github.com/edenhill/kcat>`_ (formerly known as kafkacat) is a generic non-JVM producer and consumer for Apache Kafka. 
+
+It can be used to produce and consume records to Apache Kafka topics as well as to list service configurations.
+
+
+Install kcat
+-------------
+
+kcat is an open source tool available from GitHub at https://github.com/edenhill/kcat. Installation instructions are provided in the repository README.
+
+Retrieve Aiven for Kafka SSL certificate files
+----------------------------------------------
+
+Aiven for Apache Kafka by default enables TLS security. 
+The certificates can be manually downloaded from the service overview page in the Aiven console, or via the :ref:`dedicated Aiven CLI command <avn_service_user_creds_download>`.
+
+Setup a kcat configuration file
+-------------------------------
+
+While kcat accepts all connection configuration parameters in the command line, using a configuration file helps minimising the code needed for any sequent calls.
```

Comment:
"sequent" seem to be a rarely used word, "following" or "subsequent" would sound more natural

Reply:
moved to following